lib/Controller/ChatController.php

```diff
@@ -417,6 +417,8 @@ public function receiveMessages(int $lookIntoFuture,
 			$comments = $this->chatManager->getHistory($this->room, $lastKnownMessageId, $limit, (bool) $includeLastKnown);
 		}
 
+		$comments = $this->chatManager->filterHistorySince($this->room, $comments, $attendee);
```

Comment:
**Comment:**
The filtering like this is a bit sad. It means we potentially load a lot of data from the database which we then throw away.
But I guess adding it to the comments API then creates issues with index usage, etc. So let's keep it for now.

Reply:
Will be a bit complicated to refactor the comments API.
This was the simplest way that I found. I filtered the comments before all the others processing, I think that this won't impact the performance because only will occur in the end of history.